src/main/java/nl/ordina/jobcrawler/service/VacancyStarter.java

```diff
@@ -73,4 +69,25 @@ public void scrape() throws IOException {
         log.info("Finished scraping");
         allVacancies.clear();
     }
+
+    @Scheduled(cron = "0 30 11,17 * * *") // Runs two times a day. At 11.30am and 5.30pm.
+    public void deleteNonExistingVacancies() {
+        log.info("CRON Scheduled -- Started deleting non-existing jobs");
+        List<Vacancy> allVacancies = vacancyService.getAllVacancies();
+        List<Vacancy> vacanciesToDelete = allVacancies.stream()
+                .filter(vacancy -> {
+                    if (vacancy.getBroker().equals("Yacht"))
```

Comment:
It's best practice to switch this equals statement around.
Use `"Yacht".equals(vacancy.getBroker())`.
The reasoning behind this is that you if `vacancy.getBroker()` returns null, this code will still work.

Reply:
done